Entities/Special/CTF/FlagBase.as

```diff
@@ -77,6 +77,16 @@ void onTick(CBlob@ this)
 					this.SendCommand(this.getCommandID(flag_return));
 					b.Untag("return"); //local
 
+					this.server_AttachTo(b, "FLAG");
+					b.SetFacingLeft(this.isFacingLeft());
+				}
+				if (b.hasTag("stalemate_return"))
+				{
+					//sync tag, flag can play sounds
+					b.server_DetachAll();
+					this.SendCommand(this.getCommandID(flag_return));
```

Comment:
this command will be sent by all players if this code is not only run on the server, just double-check pls
- if it _is_ just for server, there's no need to sync the tag as the sound can be/(is?) done in the cmd
- if it's not then the cmd should only be sent by the server at least (and again, sound would probably be best in the cmd hook)

Reply:
Yeah, it is server side, and no, it doesn't actually sync anything? That block is literally just a copy of the one above, I forgot to remove the comment xP